### DISCUSSION TITLE
Bouton à cocher pour que les référents lycées reçoivent les mails que reçoivent les intervenants

### DISCUSSION THREAD

**Discussion by MagaliMoserNoriaIRL** [unistra/esup-immersup-demandes#10] [6- Administration]
Pour les lycées proposant des immersions : bouton à cocher pour que le référent lycée reçoive également les mails de rappel d’inscription que reçoivent les intervenants, comme pour les référents structure au niveau des universités

**Comment by efinkbeiner5** [comment]
Ajout d'un paramétrage pour le référent lycée
Modification du batch d'envoi des mails
Chiffrage : faible

**Comment by efinkbeiner5** [comment]
Fait dans la version 4 en 2024
